app/scripts/app.js

```diff
@@ -123,7 +124,13 @@ angular
         controller: 'ImagesController'
       })
       .when('/project/:project/browse/images/:image', {
-        templateUrl: 'views/browse/image.html',
+        templateUrl: function(params) {
+          if (params.image && params.image.indexOf(':') === -1) {
```

Comment:
See https://github.com/openshift/origin-web-console/pull/235#issuecomment-235908594
would rather not have to do this